typings/index.d.ts

```diff
@@ -85,18 +72,20 @@ export interface ControllerStateAndHelpers {
     getItemProps: (options: GetItemPropsOptions) => any;
 
     // actions
-    openMenu: () => void;
-    closeMenu: () => void;
-    toggleMenu: () => void;
-    selectItem: (item: any) => void;
-    selectItemAtIndex: (index: number) => void;
-    selectHighlightedItem: (index: number) => void;
-    setHighlightedItem: (index: number) => void;
-    clearSelection: () => void;
+    openMenu: (cb: CB) => void;
+    closeMenu: (cb: CB) => void;
+    toggleMenu: (cb: CB) => void;
+    selectItem: (item: any, otherStateToSet: any, cb: CB) => void;
+    selectItemAtIndex: (index: number, otherStateToSet: any, cb: CB) => void;
+    selectHighlightedItem: (otherStateToSet: any, cb: CB) => void;
+    setHighlightedItem: (index: number, otherStateToSet: any, cb: CB) => void;
```

Comment:
`otherStateToSet` should be optional and there is no `cb` argument to this function.

Reply:
Are you talking about `setHighlightedItem` here? If so, I think that, first of all, this must be a typo, as I don't see any `setHighlightedItem` in the docs. However, there is `setHighlightedIndex`. If that is the case, I believe it does have a `cb` argument.

Reply:
Hmm, I was talking about `setHighlightedItem`, but now I don't see it anywhere in the repo except in the TypeScript definitions, so I think you can remove this line.
As for the line above, with `selectHighlightedItem`, you're right. `otherStateToSet` should be optional and there is also an optional `cb` argument.